core/keyserver/keyserver.go

```diff
@@ -76,7 +76,8 @@ type MutationLogs interface {
 	// TODO(gbelvin): Create a batch level object to make it clear that this a batch of updates.
 	Send(ctx context.Context, directoryID string, mutation ...*pb.EntryUpdate) (*WriteWatermark, error)
 	// ReadLog returns the messages in the (low, high] range stored in the
-	// specified log. ReadLog always returns complete units of the original
+	// specified log. Low and high are measured in milliseconds since the
```

Comment:
Is this API locked in? If not, I strongly recommend using a Time object instead of int64. In my experience types > comments for APIs.

Reply:
We discussed in person, but I recommend doing the migration to time.Time and then revisiting whether this PR is needed.

Reply:
Could I suggest keeping this API change for a new PR? 
The change touches quite a bit

Reply:
The changes in this PR are all related to the granularity of the time unit, and changing to time.Time will affect all of the same lines. If you feel strongly about this then we can make this change first, but I'm concerned that with this approach silly mistakes can happen (e.g. this comment says milliseconds but it should say microseconds). This wouldn't apply if we used time.Time.